runtime/src/message_processor.rs

```diff
@@ -34,6 +36,12 @@ impl ::solana_frozen_abi::abi_example::AbiExample for MessageProcessor {
     }
 }
 
+/// bprumo TODO: doc
+#[derive(Debug, Eq, PartialEq)]
+pub struct ProcessMessageResult {
```

Comment:
Could this just be a usize/u64?

Reply:
Yes! I tend to like types for their self-documenting-ness, and also to make it easier to add/remove fields while prototyping. Additionally, when a function doesn't naturally return the values I want, I find it helps to have a struct with named fields. I.e. this case, I would say it's not obvious that a call to `process_message()` should return the number of new bytes added to the accounts data. I can remove this too!

Reply:
Yeah, I could go either way depending on the case, down the leave it in